indico/modules/categories/blueprint.py

```diff
@@ -18,24 +18,28 @@
 
 from indico.modules.categories.controllers.display import RHCategoryStatistics, RHCategoryIcon, RHCategoryLogo
 from indico.modules.categories.controllers.management import (RHCreateCategory, RHDeleteCategory,
-                                                              RHManageCategoryContent, RHManageCategoryIcon,
-                                                              RHManageCategoryLogo, RHManageCategoryProtection,
-                                                              RHManageCategorySettings, RHSortSubcategories)
+                                                              RHDeleteSubcategories, RHManageCategoryContent,
+                                                              RHManageCategoryIcon, RHManageCategoryLogo,
+                                                              RHManageCategoryProtection, RHManageCategorySettings,
+                                                              RHSortSubcategories)
 from indico.web.flask.wrappers import IndicoBlueprint
 
 
 _bp = IndicoBlueprint('categories', __name__, template_folder='templates', virtual_template_folder='categories',
                       url_prefix='/category/<int:category_id>')
 
-# Management
+# Category management
 _bp.add_url_rule('/manage/', 'manage_content', RHManageCategoryContent)
 _bp.add_url_rule('/manage/delete', 'delete', RHDeleteCategory, methods=('POST',))
 _bp.add_url_rule('/manage/icon', 'manage_icon', RHManageCategoryIcon, methods=('POST', 'DELETE'))
 _bp.add_url_rule('/manage/logo', 'manage_logo', RHManageCategoryLogo, methods=('POST', 'DELETE'))
 _bp.add_url_rule('/manage/protection', 'manage_protection', RHManageCategoryProtection, methods=('GET', 'POST'))
 _bp.add_url_rule('/manage/settings', 'manage_settings', RHManageCategorySettings, methods=('POST', 'GET'))
-_bp.add_url_rule('/manage/sort-subcategories', 'sort_subcategories', RHSortSubcategories, methods=('POST',))
+
+# Subcategories management
 _bp.add_url_rule('/manage/create-subcategory', 'create_subcategory', RHCreateCategory, methods=('GET', 'POST'))
```

Comment:
/manage/subcategories/create
    /manage/subcategories/delete
    /manage/subcategories/sort

Seems more consistent now that we have that many subcategory-related endpoints.